package.json

```diff
@@ -45,18 +45,15 @@
     "react-dom": "17.0.2",
     "react-helmet": "^6.1.0",
     "react-i18next": "^11.12.0",
-    "react-redux": "7.2.4",
     "react-router-dom": "5.2.0",
     "react-virtualized": "^9.22.3",
-    "redux": "4.1.0",
-    "redux-logger": "3.0.6",
-    "redux-promise-middleware": "6.1.2",
```

Comment:
@karelhala This seems a little aggressive to modify their redux usage. It's fair to remove it because they can't really use Redux because they don't own the context. But I suspect you just pulled a feature out from under them 🤔 

@rohitkrai03 what's the use-case for your redux usage? Can you guys pivot to a React Context? That's a supported extension type and should persist between routes.

Reply:
I've looked trough the code and currently redux is used to show notification messages. Until we have full support of redux I don't think we can keep these in.